src/components/Footer/_footer.scss

```diff
@@ -0,0 +1,193 @@
+//@todo this is not fully complete and it need bit of refactoring as well.
+@mixin animated-underline($color: var(--colour-white)) {
+  display: inline-block;
+  position: relative;
+
+  &:before {
+    background-color: $color;
+    bottom: 0;
+    content: '';
+    height: var(--space-md) / 8;
+    left: 0;
+    position: absolute;
+    transform: scaleX(0);
+    transition: all 0.3s ease-in-out;
+    visibility: hidden;
+    width: 100%;
+  }
+
+  &:hover:before {
+    transform: scaleX(1);
+    visibility: visible;
+  }
+}
+
+footer {
+  background: var(--colour-blue-80);
+  color: var(--colour-white);
+  display: flex;
+}
+
+.footer__mission-statement {
+  font-family: var(--font-secondary);
+  margin: 0;
+  padding: var(--space-xl) 0;
+  text-align: center;
+
+  @include mq(md) {
+    font-size: var(--text-md);
+  }
+
+  @include mq(sm) {
+    padding: calc(6 * var(--space-md));
+  }
+
+  @include mq(md) {
+    padding: calc(8 * var(--space-md));
+  }
+}
+
+.grid-container,
+%grid-container {
+  max-width: 1344px;
+  margin-left: 5%;
+  margin-right: 5%;
+
+
+  @include mq(xxl) {
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+
+.footer__container {
+  @extend %grid-container;
+}
+
+.footer-nav__list {
+  font-size: var(--text-sm);
+  line-height: var(--heading-line-height);
+  margin: 0;
+  padding: 0;
+}
+
+.footer-nav__list--grid {
+  @include mq(sm) {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 -10px;
+
+    > .footer-nav__item {
+      margin: 0 10px;
+      width: calc(25% - #{10px});
```

Comment:
should probably be  `width: calc(25% - #{20px});` to counter the horizontal margins on left AND right